### DISCUSSION TITLE
After cobbler2.8.6 is installed, it keeps looping and installing, even if it is configured not to loop, it does not take effect

### DISCUSSION THREAD

**Discussion by 1392273211** [cobbler/cobbler#3784] [Q&A]
After cobbler2.8.6 is installed, it keeps looping and installing, even if it is configured not to loop, it does not take effect

**Comment by opoplawski** [comment]
Is the nopxe trigger being called from the kickstart %post section? Are there any errors in the anaconda, cobbler, or web server logs?
